subsys/bluetooth/host/l2cap.c

```diff
@@ -3138,7 +3138,24 @@ static int bt_l2cap_dyn_chan_send(struct bt_l2cap_le_chan *le_chan, struct net_b
 		return -ENOBUFS;
 	}
 
-	/* Prepend SDU "header" */
+	/* Prepend SDU "header".
+	 *
+	 * L2CAP LE CoC SDUs are segmented into PDUs and sent over so-called
+	 * K-frames that each have their own L2CAP header (ie channel, PDU
+	 * length).
+	 *
+	 * The SDU header is right before the data that will be segmented and is
+	 * only present in the first segment/PDU. Here's an example:
+	 *
+	 * Sent data payload of 50 bytes over channel 0x4040 with MPS of 30 bytes:
+	 * First PDU / segment:
+	 * | l2 header                   | sdu header   | payload          |
```

Comment:
Add line something like:
```
	 * | l2 header                     | "PDU payload"            |
```
PDU payload covers sdu header and app payload chunk.

Reply:
done and moved to prev PR